code/modules/surgery/surgery_step.dm

```diff
@@ -67,6 +67,10 @@
 	var/fail_prob = 0//100 - fail_prob = success_prob
 	var/advance = FALSE
 
+	if(surgery.organ_to_manipulate && !target.getorganslot(surgery.organ_to_manipulate))
+		to_chat(user, span_warning("[target] seems to be missing the organ necessary to complete this surgery!"))
+		return FALSE
```

Comment:
Looking at the rest of the code, is there a reason why this is in `initiate` and not `try_op` somewhere?